test/language/expressions/optional-chaining/call-expression-optional-chain.js

```diff
@@ -0,0 +1,17 @@
+/*---
+esid: pending
+info: |
+  optional chain on call expression
+description: >
```

Comment:
invert usage of info and description. Description is the small line, info is the large chunk.

You may keep the block characters `info: |` (pipe) and `description: >` (greater than)